platform/security/core/security-core-impl/src/main/java/ddf/security/config/impl/ConfigurationSecurityLogger.java

```diff
@@ -56,9 +56,11 @@ public void configurationEvent(ConfigurationEvent event) {
         // there was no subject change was caused by an update to the config file on the filesystem
         SecurityLogger.auditWarn("Configuration {} via filesystem: {}", type, updatedConfiguration);
       }
+    } catch (VirtualMachineError vme) {
+      throw vme;
     } catch (Throwable e) {
-      LOGGER.error("Error auditing config update for {}", event.getPid(), e);
-      SecurityLogger.audit("Error auditing config update for {}", event.getPid());
+      LOGGER.error("Error auditing config update for " + event.getPid(), e);
+      SecurityLogger.audit("Error auditing config update for ", event.getPid());
```

Comment:
❗️ Line 62 is correct with your change, as you can't intermix interpolation params and exceptions. However, your change to line 63 is incorrect as you've removed the `{}` and not changed to string concatenation. I would probably just leave the `SecurityLogger` call as it was and only fix the `LOGGER` call. 

If the inconsistency worries you (that someone may misread in the future and break it), you can always leave a short in-line comment explaining the difference.

Reply:
Oops, fixed!